core-tests/src/test/java/org/mule/runtime/core/el/mvel/MVELExpressionLanguageTestCase.java

```diff
@@ -264,11 +255,7 @@ public void appTakesPrecedenceOverEverything() throws Exception {
   @Test
   public void messageTakesPrecedenceOverEverything() throws Exception {
     mvel.setAliases(Collections.singletonMap("message", "'other1'"));
-    FlowConstruct flowConstruct = MuleTestUtils.getTestFlow(muleContext);
-    Event event = Event.builder(DefaultEventContext.create(flowConstruct, TEST_CONNECTOR))
-        .message(InternalMessage.of(""))
-        .addVariable("message", "other2")
-        .build();
+    Event event = eventBuilder().message(InternalMessage.of("")).addVariable("message", "other2").build();
```

Comment:
If the message is often not important in tests, then can we make it optional and use a default/null message instead?  This may mean a slightly different builder for tests though, so maybe not best idea..  Alternatively a NULL_MESSAGE or EMPTY_MESSAGE constant could help, and not require extending builder in test infrasturucture.